lib/brainstem/api_docs.rb

```diff
@@ -0,0 +1,43 @@
+require 'active_support/configurable'
+
+module Brainstem
+  module ApiDocs
+    include ActiveSupport::Configurable
+
+    config_accessor(:base_path) do
+      File.expand_path("./brainstem_docs")
+    end
+
+    config_accessor(:controller_filename_pattern) do
+      File.join("controllers", "{{name}}_controller.{{extension}}")
```

Comment:
This style as in the moustaches? It's used inside of `suggested_filename` methods inside any of the non-collection domain wrappers (`endpoint.rb`, `controller.rb`, `presenter.rb`). It's still a bit kludgy IMO but being able to specify the format of it should add some flexibility — in fact, one thing that might be nice to add would be to add a separate `{{ api_version_path }}` that could be included or removed so that controllers aren't by default output into `/api/v1/`.